src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -147,6 +147,11 @@ def optional_key_ref_to_entity_ref(key_ref: Optional[KeyReference]) -> Optional[
             for link in links
         )
 
+    def _validate_drs_uri(self, drs_uri: Optional[str]) -> None:
+        # Some AnVIL snapshots contain DRS URI's that use "compact identifiers"
+        # that do not conform to standard URL syntax.
+        pass
```

Comment:
There valid according to some specification, however harmful that specification may be, so you should be able to validate conformance with that specification.